### DISCUSSION TITLE
add 32 bits (x86) support

### DISCUSSION THREAD

**Discussion by pirocudo3000** [ntsc-rs/ntsc-rs#96] [Ideas]
i have a 32 bits laptop and i wanted to use ntsc rs, unfortunately it wont work bc it is only for 64 bits :(

**Comment by valadaptive** [comment]
How old is your laptop, and what processor does it have? Anything from 2006 or newer (2012 if it's an Intel Atom) should support 64-bit as long as you install a 64-bit version of Windows.

**Reply by smfthe1199** [reply]
Now this is an idea, or suggestion, not an issue, or problem. You should **make** a 32-but version of this app that should support anything before 2006 _that is_!

**Reply by valadaptive** [reply]
Why? ntsc-rs *should* build successfully on 32-bit platforms, but providing official 32-bit builds would be an additional maintenance burden, especially when hardware that old probably can't run ntsc-rs for other reasons (lack of graphics drivers, unsupported Windows versions, etc).

**Reply by smfthe1199** [reply]
Oh hmm..

**Reply by smfthe1199** [reply]
This is probably the reason why ntsc-rs doesn't run on my old PC, it runs on a newer PC/Laptop for that matter, hence that's why, but adding 32-bit support to this?

And on top of that, it **won't certainly work**, especially, I had to suffer before migrating!

**Reply by valadaptive** [reply]
How old is your old PC? Anything with an Intel Core 2 or newer supports 64-bit.

**Reply by Degamisu** [reply]
> This is probably the reason why ntsc-rs doesn't run on my old PC, it runs on a newer PC/Laptop for that matter, hence that's why, but adding 32-bit support to this?

if you have a new computer, why not use that? the old computer seems redundant in this situation?

**Reply by smfthe1199** [reply]
Now this **isn't** my idea, this is @pirocudo3000 's idea!

**Reply by smfthe1199** [reply]
@valadaptive remember when I opened an issue about me trying to run ntsc-rs, but didn't work?

This is because of hardware issue, even though, my old PC is 64-bit, but doesn't have the features of my other device, my Laptop from 2015 can run this!

**Reply by smfthe1199** [reply]
> How old is your old PC? Anything with an Intel Core 2 or newer supports 64-bit.

Btw, answering your question.

**If you ask me:** Yes, I do have an Intel Core 2+ all into my 2015 laptop that originally ran Windows 8.1 before upgraded to Windows 10

**If you ask the original poster:** Then ask him/her!

**Reply by valadaptive** [reply]
The reason ntsc-rs couldn't run on your old PC was because of an unsupported graphics driver. Nothing to do with 32-bit support.

**Reply by Degamisu** [reply]
> The reason ntsc-rs couldn't run on your old PC was because of an unsupported graphics driver. Nothing to do with 32-bit support.

It could also be an unsupported operating system. But an unsupported graphics driver may be the case. so @valadaptive , what is the current say on this? or are we still deciding?

**Reply by smfthe1199** [reply]
Yeah! If we upgrade our graphic cards, maybe this should work!

**Reply by valadaptive** [reply]
> It could also be an unsupported operating system. But an unsupported graphics driver may be the case. so @valadaptive , what is the current say on this? or are we still deciding?

I guess you could put it either way. Their graphics driver didn't support Windows 10.

**Reply by Degamisu** [reply]
> Yeah! If we upgrade our graphic cards, maybe this should work!

Well not necessarily the hardware itself, unless it was discontinued a long time ago, The graphics drivers would most likely need updating. I could be wrong if it was like, I don't know, and FX 600 😆 .
